stdlib/atomic.mli

```diff
@@ -17,6 +17,84 @@
 
 (** Atomic references.
 
+    A basic use case is to have global counters that are updated in a
+    thread-safe way:
+
+    {[
+    (* our counter *)
+    let count_bytes_read = Atomic.make 0;;
```

Comment:
`;;` is not idiomatic OCaml -- and not necessary for toplevel copyability either. Can you explain why you want to keep them around?

Reply:
They are useful if  you send phrases to the toplevel one by one. I can remove them.

Reply:
Unless you have a strong opinion, I would remove them. I think it's better to show the code as we would actually write it in a source file. (Note: Tuareg has a shortcut to send a sentence to the toplevel, which adds `;;` automatically, doesn't your editor support something like this?)